bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -1,321 +1,733 @@
-					organizer_email_address_a: calendarValue?.organizer?.emailAddress?.address ?? null,
-					organizer_email_address_n: calendarValue?.organizer?.emailAddress?.name ?? null,
-					original_start_time_zone: calendarValue?.originalStartTimeZone ?? null,
-					series_master_id: calendarValue?.seriesMasterId ?? null,
-					start_date_time: calendarValue?.start?.dateTime ?? null,
-					subject: calendarValue?.subject ?? null
-				});
-				for (const attendees of calendarValue.attendees) {
-					dataStore.save('calender_view_attendees', {
-						unique_id: uuid.v4(),
-						parent_i_cal_u_id: calendarValue?.iCalUId ?? null,
-						root_i_cal_u_id: calendarValue?.iCalUId ?? null,
-						email_address_address: attendees?.emailAddress?.address ?? null,
-						email_address_name: attendees?.emailAddress?.name ?? null,
-						type: attendees?.type ?? null
-					});
-				}
-			}
-		}
-	}
+async function myEvents(client, dataStore, userIds) {
+    for (const id of userIds) {
+        const myEventRequest = await client.fetch(`/v1.0/users/${id}/calendar/events?$top=${pageSize}`);
+        if (!myEventRequest.ok && myEventRequest.status != 404) {
+            throw new Error(`Events sync failed ${myEventRequest.status}:${myEventRequest.statusText}`);
+        }
+        const myEventResponse = await myEventRequest.json();
+        if (myEventRequest.ok) {
+            for (const myEventsValue of myEventResponse.value) {
+                dataStore.save('my_events', {
+                    i_cal_u_id: myEventsValue?.iCalUId ?? null,
+                    id: myEventsValue?.id ?? null,
+                    recurrence_pattern_day_of: myEventsValue?.recurrence?.pattern?.dayOfMonth ?? null,
+                    recurrence_pattern_type: myEventsValue?.recurrence?.pattern?.type ?? null,
+                    recurrence_range_end_date: myEventsValue?.recurrence?.range?.endDate ?? null
+                });
+            }
+        }
+    }
 }
 
-async function myEvents(client, dataStore, userId) {
-	for (const id of userId) {
-		const myEventRequest = await client.fetch(`/v1.0/users/${id}/calendar/events?$top=${pageSize}`);
-		if (!myEventRequest.ok && myEventRequest.status != 404) {
-			throw new Error(`Events sync failed ${myEventRequest.status}:${myEventRequest.statusText}`);
-		}
-		const myEventResponse = await myEventRequest.json();
-		if (myEventRequest.ok) {
-			for (const myEventsValue of myEventResponse.value) {
-				dataStore.save('my_events', {
-					i_cal_u_id: myEventsValue?.iCalUId ?? null,
-					id: myEventsValue?.id ?? null,
-					recurrence_pattern_day_of: myEventsValue?.recurrence?.pattern?.dayOfMonth ?? null,
-					recurrence_pattern_type: myEventsValue?.recurrence?.pattern?.type ?? null,
-					recurrence_range_end_date: myEventsValue?.recurrence?.range?.endDate ?? null
-				});
-			}
-		}
-	}
+
+
+
+async function editRecurringEventwithCurrentTimezone(param) {
+    const { client, store, actionParameters } = param
```

Comment:
Does this `store` really works? `store` is not a inbuilt library, `dataStore` is a inbuilt library.
Please double check